Doc/library/os.rst

```diff
@@ -569,6 +569,32 @@ process and user.
       See the documentation for :func:`getgroups` for cases where it may not
       return the same group list set by calling setgroups().
 
+.. function:: setns(fd, nstype=0)
+
+   Reassociate thread with a namespace, see the :manpage:`setns(2)` man page for more details.
+
+   If *fd* refers to a ``/proc/[pid]/ns/`` link, ``setns()`` reassociates the
+   calling thread with the namespace associated with that link, subject to any
+   constraints imposed by the *nstype* argument (or any if ``0``).
```

Comment:
Unclear what ```(or any if `0`)``` means here. I assume this is saying that there are no constraints if `nstype` is passed as zero?

If so, then something like ```(a *nstype* of `0` means no constraints)``` would be clearer to me.